### DISCUSSION TITLE
Where to start reading?

### DISCUSSION THREAD

**Discussion by Saswat7101** [hackclubiter/onEditor#8] [Q&A] [answered]
I was searching about how to make online compilers and all ........ but upon reading various articles i found that most of the things went over my head ....... so is there any resource or place where we can begin reading so that we get some basic ideas relating to this

**Comment by saswatsamal** [comment, marked as answer]
I would like to suggest you kindly check the course by Standford which teaches about the compilers.
[Link](https://www.edx.org/course/compilers)

**Reply by LoopGlitch26** [reply]
Very nice resource @saswatsamal 
Here's a few more resources regarding Compilers Design that might be helpful
[Link](https://youtube.com/playlist?list=PLEbnTDJUr_IcPtUXFy2b1sGRPsLFMghhS)

**Comment by Saswat7101** [comment]
Ohkk ty guys

**Comment by LoopGlitch26** [comment]
I also found [this](https://github.com/lawrancej/CompilerDesign/tree/master/textbook) which can be an interesting place to start reading and get an idea about Compiler Design

**Reply by saswatsamal** [reply]
Yes, this repository gives an insight about the CD.

**Comment by Saswat7101** [comment]
I have heard that [NPTEL](https://nptel.ac.in/courses/106/105/106105190/) lectures are also good for any purpose
